src/datumaro/cli/commands/explore.py

```diff
@@ -125,20 +149,29 @@ def explore_command(args):
         project.working_tree.config.update(build_tree.config)
         project.working_tree.save()
 
-    # Get query datasetitem through query path
-    if osp.exists(args.query):
-        query_datasetitem = None
-        for dataset in source_datasets:
-            try:
-                query_datasetitem = dataset.get_datasetitem_by_path(args.query)
-            except Exception:
-                continue
-            if not query_datasetitem:
-                break
-    else:
-        query_datasetitem = args.query
-
-    results = explorer.explore_topk(query_datasetitem, args.topk)
+    if args.query_img_path:
+        querys = check_and_convert_to_list(args.query_img_path)
+        query_datasetitems = []
+        for query_ in querys:
+            query_datasetitem = DatasetItem(
+                id=random.randrange(100, 1000), media=Image.from_file(query_)
```

Comment:
Basically, `id` type should be `str`. If you want to generate a random unique id, please use `uuid4`: https://docs.python.org/ko/3/library/uuid.html.